packages/smooth_app/lib/pages/user_management/login_page.dart

```diff
@@ -90,211 +90,251 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
+                          crossAxisAlignment: CrossAxisAlignment.center,
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          mainAxisSize: MainAxisSize.min,
+                          children: <Widget>[
+                            SvgPicture.asset(
+                              'assets/preferences/login.svg',
+                              height: MediaQuery.of(context).size.height * .15,
+                            ),
+                            const SizedBox(
+                              height: LARGE_SPACE,
+                            ),
+                            Text(
+                              appLocalizations.sign_in_text,
+                              textAlign: TextAlign.center,
+                              style: theme.textTheme.headline1?.copyWith(
+                                fontSize: 19.0,
+                                height: 1.26,
+                                fontWeight: FontWeight.w600,
+                                color: theme.colorScheme.onSurface,
+                              ),
+                            ),
 
-                      const SizedBox(
-                        height: LARGE_SPACE,
-                      ),
+                            const SizedBox(
+                              height: LARGE_SPACE * 2,
+                            ),
 
-                      //Password
-                      SmoothTextFormField(
-                        type: TextFieldTypes.PASSWORD,
-                        controller: passwordController,
-                        hintText: appLocalizations.password,
-                        textColor: _customGrey,
-                        backgroundColor: _textFieldBackgroundColor,
-                        prefixIcon: const Icon(Icons.vpn_key),
-                        enabled: !_runningQuery,
-                        textInputAction: TextInputAction.done,
-                        // Hides the keyboard
-                        autofillHints: const <String>[
-                          AutofillHints.password,
-                        ],
-                        validator: (String? value) {
-                          if (value == null || value.isEmpty) {
-                            return appLocalizations
-                                .login_page_password_error_empty;
-                          }
-                          return null;
-                        },
-                      ),
+                            if (_wrongCredentials) ...<Widget>[
+                              SmoothCard(
+                                padding: const EdgeInsets.all(10.0),
+                                color: Colors.red,
+                                child: Text(
+                                    appLocalizations.incorrect_credentials),
+                              ),
+                              const Spacer(
+                                flex: 1,
+                              )
```

Comment:
This is probably the cause of the merge conflicts this also has the be a SizedBox, this spacer caused the login page to break when logging in with wrong credentials